modules/ossm-rn-fixed-issues.adoc

```diff
@@ -31,6 +33,8 @@ The following issues been resolved in the current release:
 
 * link:https://issues.redhat.com/browse/OSSM-287[OSSM-287] In the Kiali console there are no traces being displayed on the Graph Service.
 
+* link:https://issues.redhat.com/browse/MAISTRA-2631[MAISTRA-2631] WASM feature is not working because podman is failing due to nsenter binary not present. {ProductName} generates the following error message: `Error: error configuring CNI network plugin exec: "nsenter": executable file not found in $PATH`. The container image now contains nsenter and WASM works as expected.
```

Comment:
I'd add an article here: "The WASM feature is not working ..."

Reply:
The last part of the sentence also sounds a little bit unlike a sentence. Maybe something like "due to the nsenter binary not being present" could be better.

Reply:
"is not working" sounds a bit casual, in my opinion.

Reply:
Thanks for these suggestions, Eva-Lotte. I opted to retain this phrasing given the deadline, but will keep the casual tone in mind going forward.